Ways to connect to cluster using kn (#4095)
knative/docs: docs/client/README.md

## Connecting CLI tools to your cluster

After you have installed `kubectl` or `kn`, these tools will search for the `kubeconfig` file of your cluster in the default location of `$HOME/.kube/config`, and will use this file to connect to the cluster. A `kubeconfig` file is usually automatically created when you create a Kubernetes cluster.

You can also set the environment variable `$KUBECONFIG`, and point it to the kubeconfig file.

> Using the `kn` CLI, we can specify following options to connect to the cluster.

Reviewer: ```suggestion
Using the `kn` CLI, you can specify the following options to connect to the cluster:

```

> - `--kubeconfig` option is equivalent to setting the `$KUBECONFIG` environment variable and can be used to point to the kubeconfig file.

Reviewer: ```suggestion
- `--kubeconfig`: use this option to point to the `kubeconfig` file. This is equivalent to setting the `$KUBECONFIG` environment variable.
```

> - `--context` can be used to specify the name of a context from the existing kubeconfig. (One of the output list of: `kubectl config get-contexts`)

Reviewer: ```suggestion
- `--context`: use this option to specify the name of a context from the existing `kubeconfig` file. Use one of the contexts from the output of `kubectl config get-contexts`.
```
